sdk/src/client/message_interface/response.rs

```diff
@@ -94,6 +95,12 @@ pub enum Response {
     /// - [`SignatureUnlock`](crate::message_interface::Message::SignatureUnlock)
     SignatureUnlock(UnlockDto),
     /// Response for:
+    /// - [`SignEd25519`](crate::message_interface::Message::SignEd25519)
+    Ed25519Signature(Ed25519SignatureDto),
+    /// Response for:
+    /// - [`VerifyEd25519Signature`](crate::message_interface::Message::VerifyEd25519Signature)
+    ValidSignature(bool),
```

Comment:
Can't really think of something nice that would use verify instead. IsSignatureVerified ? VerifiedSignature ? ... ?

Reply:
An alternative would be to just return `Bool(bool)`, we can use that at multiple places

Reply:
Yeah that's actually pretty nice 👍🏻